enterprise/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/ide/WildflyJ2eePlatformFactory.java

```diff
@@ -177,6 +177,9 @@ public Set<org.netbeans.api.j2ee.core.Profile> getSupportedProfiles() {
             if (this.properties.getServerVersion().compareToIgnoreUpdate(WildflyPluginUtils.EAP_7_0) >= 0) {
                 return Collections.unmodifiableSet(WILDFLY_PROFILES);
             }
+            if (this.properties.getServerVersion().compareToIgnoreUpdate(WildflyPluginUtils.EAP_8_0) >= 0) {
```

Comment:
This code can't evaluate to true. `compareToIgnoreUpdate` is called with a lower version directly before. Both check for greater or equals. So an 8.0.0 JBboss should be detected by line 177.

Reply:
Removed